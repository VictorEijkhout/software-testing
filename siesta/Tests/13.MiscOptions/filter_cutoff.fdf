%include ../base_wat_smallbox.fdf
SystemLabel     filter_cutoff

%block ChemicalSpeciesLabel
 1  8  O
 2  1  H
%endblock ChemicalSpeciesLabel

FilterCutoff 150.0 Ry
MeshCutoff   200.0 Ry

