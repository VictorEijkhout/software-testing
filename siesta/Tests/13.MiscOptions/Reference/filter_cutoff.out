Siesta Version  : 5.0.0-alpha-123-gfb8fd7d02
Architecture    : ----
Compiler version: GNU-11.4.0
Compiler flags  : -fallow-argument-mismatch;-O3 -march=native
PP flags        : ----
Libraries       : ----
Parallelisations: none
GEMM3M support
NetCDF support
NetCDF-4 support
Lua support

Runtime information:
* Directory : /home/fnpedron/Siesta/siesta-dev/Tests/13.MiscOptions/work/filter_cutoff
* Running in serial mode.
>> Start of run:  28-SEP-2023  15:40:25

                           *********************** 
                           *  WELCOME TO SIESTA  * 
                           *********************** 

reinit: Reading from standard input
reinit: Dumping input in INPUT_TMP.18943
************************** Dump of input data file ****************************
%include ../../basejob_2.fdf
SystemLabel     filter_cutoff
%block ChemicalSpeciesLabel
 1  8  O
 2  1  H
%endblock ChemicalSpeciesLabel
FilterCutoff 150.0 Ry
MeshCutoff   200.0 Ry
************************** End of input data file *****************************

reinit: -----------------------------------------------------------------------
reinit: System Name: 2 Water molecules
reinit: -----------------------------------------------------------------------
reinit: System Label: filter_cutoff
reinit: -----------------------------------------------------------------------

initatom: Reading input for the pseudopotentials and atomic orbitals ----------
Species number:   1 Atomic number:    8 Label: O
Species number:   2 Atomic number:    1 Label: H


---- Processing specs for species: O
Ground state valence configuration:   2s02  2p04
Reading pseudopotential information in formatted form from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/O.psf

---- Processing specs for species: H
Ground state valence configuration:   1s01
Reading pseudopotential information in formatted form from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/H.psf

---- Pseudopotential check for O 

Pseudized shells:
2s( 2.00) rc: 1.14
2p( 4.00) rc: 1.14
3d( 0.00) rc: 1.14
4f( 0.00) rc: 1.14
Valence configuration for ps generation: (assumed as above)

---- Pseudopotential check for H 

Pseudized shells:
1s( 1.00) rc: 1.25
2p( 0.00) rc: 1.25
3d( 0.00) rc: 1.25
4f( 0.00) rc: 1.25
Valence configuration for ps generation: (assumed as above)
For O, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.
For H, standard SIESTA heuristics set lmxkb to 2
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.

<basis_specs>
===============================================================================
O                    Z=   8    Mass=  15.999        Charge= 0.17977+309
Lmxo=2 Lmxkb= 3    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=2
          i=1  nzeta=1  polorb=0  (2s)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=1  Nsemic=0  Cnfigmx=2
          i=1  nzeta=1  polorb=1  (2p)  (to be polarized perturbatively)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=2  Nsemic=0  Cnfigmx=3
          i=1  nzeta=0  polorb=0  (3d)  (perturbative polarization orbital) (from 2p)
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
L=3  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Filter: Requested cutoff:  150.000000 Ry
Using PAO.SplitTailNorm
atom: Called for O                     (Z =   8)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Valence charge in pseudo generation:    6.00000

xc_check: Exchange-correlation functional:
xc_check: Ceperley-Alder
V l=0 = -2*Zval/r beyond r=  1.1366
V l=1 = -2*Zval/r beyond r=  1.1332
V l=2 = -2*Zval/r beyond r=  1.1298
V l=3 = -2*Zval/r beyond r=  1.1264
All V_l potentials equal beyond r=  1.1355
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  1.1366
Using small-core scheme (local charge) for Vlocal

VLOCAL_FROM_CHLOCAL: 99.0% of the norm of Vloc inside     33.667 Ry
VLOCAL_FROM_CHLOCAL: 99.9% of the norm of Vloc inside     76.727 Ry
atom: Maximum radius for chlocal:    1.37241
atom: Maximum radius for r*vlocal+2*Zval:    1.25469

KBgen: Kleinman-Bylander projectors: 
GHOST: No ghost state for L =  0
   l= 0   rc=  1.293209   el= -1.742414   Ekb=  9.076872   kbcos=  0.325320
GHOST: No ghost state for L =  1
   l= 1   rc=  1.284119   el= -0.676589   Ekb= -8.194485   kbcos= -0.396010
GHOST: No ghost state for L =  2
   l= 2   rc=  1.433152   el=  0.013307   Ekb= -2.071665   kbcos= -0.003690
GHOST: No ghost state for L =  3
   l= 3   rc=  1.540891   el=  0.019584   Ekb= -0.817137   kbcos= -0.000371

KBgen: Total number of Kleinman-Bylander projectors:  16
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 2s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    3.560943
                 energy =   -1.731321
                kinetic =    1.587329
    potential(screened) =   -3.318651
       potential(ionic) =  -11.260325
PAO Filter: Cutoff used:   73.50 Ry
PAO Filter: diagnostic kin energy tol:    0.003000 Ry
PAO Filter: Nominal cutoff (before filtering):   54.44 Ry
Filter:    Number of eigenfunctions  of the
           filtering kernel (total, used)= 30  8

SPLIT: Orbitals with angular momentum L= 1

SPLIT: Basis orbitals for state 2p

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    4.290924
                 energy =   -0.665565
                kinetic =    4.934810
    potential(screened) =   -5.600376
       potential(ionic) =  -13.346429
PAO Filter: Cutoff used:   73.50 Ry
PAO Filter: diagnostic kin energy tol:    0.003000 Ry
PAO Filter: Nominal cutoff (before filtering):   83.79 Ry
Filter:    Number of eigenfunctions  of the
           filtering kernel (total, used)= 34 10

POLgen: Perturbative polarization orbital with L=  2

POLgen: Polarization orbital for state 2p
Split based on tail norm

   izeta = 1
                     rc =    4.290924
                 energy =    2.236783
                kinetic =    4.278617
    potential(screened) =   -2.041834
       potential(ionic) =   -8.171412
PAO Filter: diagnostic kin energy tol:    0.003000 Ry
PAO Filter: Nominal cutoff (before filtering):  100.70 Ry
Filter:    Number of eigenfunctions  of the
           filtering kernel (total, used)= 34  9
atom: Total number of Sankey-type orbitals:  9

atm_pop: Valence configuration (for local Pseudopot. screening):
 2s( 2.00)                                                            
 2p( 4.00)                                                            
 3d( 0.00)                                                            
Vna: chval, zval:    6.00000   6.00000

Vna:  Cut-off radius for the neutral-atom potential:   4.290924
VNA Filter: diagnostic kinetic energy tol:    0.003000 Ry
VNA Filter: Nominal cutoff (before filtering):   43.86 Ry
VNA Filter: Cutoff used:   150.00 Ry
Filter:    Number of eigenfunctions  of the
           filtering kernel (total, used)= 44 15

atom: _________________________________________________________________________

<basis_specs>
===============================================================================
H                    Z=   1    Mass=  1.0080        Charge= 0.17977+309
Lmxo=1 Lmxkb= 2    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=1
          i=1  nzeta=1  polorb=1  (1s)  (to be polarized perturbatively)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=1  Nsemic=0  Cnfigmx=2
          i=1  nzeta=0  polorb=0  (2p)  (perturbative polarization orbital) (from 1s)
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Filter: Requested cutoff:  150.000000 Ry
Using PAO.SplitTailNorm
atom: Called for H                     (Z =   1)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Valence charge in pseudo generation:    1.00000

xc_check: Exchange-correlation functional:
xc_check: Ceperley-Alder
V l=0 = -2*Zval/r beyond r=  1.2384
V l=1 = -2*Zval/r beyond r=  1.2321
V l=2 = -2*Zval/r beyond r=  1.2272
All V_l potentials equal beyond r=  1.2371
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  1.2384
Using small-core scheme (local charge) for Vlocal

VLOCAL_FROM_CHLOCAL: 99.0% of the norm of Vloc inside     28.363 Ry
VLOCAL_FROM_CHLOCAL: 99.9% of the norm of Vloc inside     64.639 Ry
atom: Maximum radius for chlocal:    1.44620
atom: Maximum radius for r*vlocal+2*Zval:    1.31026

KBgen: Kleinman-Bylander projectors: 
GHOST: No ghost state for L =  0
   l= 0   rc=  1.364142   el= -0.467325   Ekb= -2.009475   kbcos= -0.336680
GHOST: No ghost state for L =  1
   l= 1   rc=  1.430269   el=  0.007562   Ekb= -0.503817   kbcos= -0.022554
GHOST: No ghost state for L =  2
   l= 2   rc=  1.469686   el=  0.013286   Ekb= -0.191752   kbcos= -0.002430

KBgen: Total number of Kleinman-Bylander projectors:   9
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 1s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    5.285354
                 energy =   -0.456120
                kinetic =    0.900445
    potential(screened) =   -1.356565
       potential(ionic) =   -1.890080
PAO Filter: Cutoff used:   73.50 Ry
PAO Filter: diagnostic kin energy tol:    0.003000 Ry
PAO Filter: Nominal cutoff (before filtering):   27.17 Ry
Filter:    Number of eigenfunctions  of the
           filtering kernel (total, used)= 39 13

POLgen: Perturbative polarization orbital with L=  1

POLgen: Polarization orbital for state 1s
Split based on tail norm

   izeta = 1
                     rc =    5.285354
                 energy =    0.642323
                kinetic =    1.282472
    potential(screened) =   -0.640149
       potential(ionic) =   -1.117275
PAO Filter: diagnostic kin energy tol:    0.003000 Ry
PAO Filter: Nominal cutoff (before filtering):   38.29 Ry
Filter:    Number of eigenfunctions  of the
           filtering kernel (total, used)= 39 12
atom: Total number of Sankey-type orbitals:  4

atm_pop: Valence configuration (for local Pseudopot. screening):
 1s( 1.00)                                                            
 2p( 0.00)                                                            
Vna: chval, zval:    1.00000   1.00000

Vna:  Cut-off radius for the neutral-atom potential:   5.285354
VNA Filter: diagnostic kinetic energy tol:    0.003000 Ry
VNA Filter: Nominal cutoff (before filtering):   33.11 Ry
VNA Filter: Cutoff used:   150.00 Ry
Filter:    Number of eigenfunctions  of the
           filtering kernel (total, used)= 52 19

atom: _________________________________________________________________________

prinput: Basis input 

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1    8 O                       # Species index, atomic number, species label
    2    1 H                       # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block PAO.Basis                 # Define Basis set
# WARNING: This information might be incomplete!
O                     2                    # Species label, number of l-shells
 n=2   0   1                         # n, l, Nzeta 
   3.561   
   1.000   
 n=2   1   1 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   4.291   
   1.000   
H                     1                    # Species label, number of l-shells
 n=1   0   1 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   5.285   
   1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

Dumping basis to NetCDF file O.ion.nc
Dumping basis to NetCDF file H.ion.nc
coor:   Atomic-coordinates input format  =     Cartesian coordinates
coor:                                          (in Angstroms)

siesta: Atomic coordinates (Bohr) and species
siesta:      0.00000   0.00000   0.00000  1        1
siesta:      1.43052   1.10738   0.00000  2        2
siesta:     -1.43052   1.10738   0.00000  2        3
siesta:      0.00000   9.44863   0.00000  1        4
siesta:      1.43052   8.34125   0.00000  2        5
siesta:     -1.43052   8.34125   0.00000  2        6

siesta: System type = slab      

initatomlists: Number of atoms, orbitals, and projectors:      6    34    68

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Spin configuration                          = none
redata: Number of spin components                   = 1
redata: Time-Reversal Symmetry                      = T
redata: Spin spiral                                 = F
redata: Long output                                 =   F
redata: Number of Atomic Species                    =        2
redata: Charge density info will appear in .RHO file
redata: Write Mulliken Pop.                         = NO
redata: Matel table size (NRTAB)                    =     1024
redata: Mesh Cutoff                                 =   200.0000 Ry
redata: Net charge of the system                    =     0.0000 |e|
redata: Min. number of SCF Iter                     =        0
redata: Max. number of SCF Iter                     =     1000
redata: SCF convergence failure will abort job
redata: SCF mix quantity                            = Hamiltonian
redata: Mix DM or H after convergence               =   F
redata: Recompute H after scf cycle                 =   F
redata: Mix DM in first SCF step                    =   T
redata: Write Pulay info on disk                    =   F
redata: New DM Occupancy tolerance                  = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks                  =     0.5000
redata: Require Harris convergence for SCF          =   F
redata: Harris energy tolerance for SCF             =     0.000100 eV
redata: Require DM convergence for SCF              =   T
redata: DM tolerance for SCF                        =     0.000100
redata: Require EDM convergence for SCF             =   F
redata: EDM tolerance for SCF                       =     0.001000 eV
redata: Require H convergence for SCF               =   T
redata: Hamiltonian tolerance for SCF               =     0.001000 eV
redata: Require (free) Energy convergence for SCF   =   F
redata: (free) Energy tolerance for SCF             =     0.000100 eV
redata: Using DFT-D3 dispersion                     =   F
redata: Using Saved Data (generic)                  =   F
redata: Use continuation files for DM               =   F
redata: Neglect nonoverlap interactions             =   F
redata: Method of Calculation                       = Diagonalization
redata: Electronic Temperature                      =   299.9863 K
redata: Fix the spin of the system                  =   F
redata: Max. number of TDED Iter                    =        1
redata: Number of TDED substeps                     =        3
redata: Dynamics option                             = Single-point calculation
mix.SCF: Pulay mixing                            = Pulay
mix.SCF:    Variant                              = stable
mix.SCF:    History steps                        = 2
mix.SCF:    Linear mixing weight                 =     0.250000
mix.SCF:    Mixing weight                        =     0.250000
mix.SCF:    SVD condition                        = 0.1000E-07
redata: Save all siesta data in one NC              =   F
redata: ***********************************************************************

%block SCF.Mixers
  Pulay
%endblock SCF.Mixers

%block SCF.Mixer.Pulay
  # Mixing method
  method pulay
  variant stable

  # Mixing options
  weight 0.2500
  weight.linear 0.2500
  history 2
%endblock SCF.Mixer.Pulay

DM_history_depth set to one: no extrapolation allowed by default for geometry relaxation
Size of DM history Fstack: 1
Total number of electrons:    16.000000
Total ionic charge:    16.000000

siesta: k-grid: Number of k-points = 1
siesta: k-point is Gamma-only

diag: Algorithm                                     = D&C
diag: Used triangular part                          = Lower
diag: Absolute tolerance                            =  0.100E-15
diag: Orthogonalization factor                      =  0.100E-05
diag: Memory factor                                 =  1.0000


ts: **************************************************************
ts: Save H and S matrices                           =    F
ts: Save DM and EDM matrices                        =    F
ts: Only save the overlap matrix S                  =    F
ts: **************************************************************

************************ Begin: TS CHECKS AND WARNINGS ************************
************************ End: TS CHECKS AND WARNINGS **************************


                     ====================================
                        Single-point calculation
                     ====================================

outcell: Unit cell vectors (Ang):
        3.000000    0.000000    0.000000
        0.000000   10.000000    0.000000
        0.000000    0.000000    5.000000

outcell: Cell vector modules (Ang)   :    3.000000   10.000000    5.000000
outcell: Cell angles (23,13,12) (deg):     90.0000     90.0000     90.0000
outcell: Cell volume (Ang**3)        :    150.0000
Gamma-point calculation with interaction between periodic images
Some features might not work optimally:
e.g. DM initialization from atomic data
<dSpData1D:S at geom step 0
  <sparsity:sparsity for geom step 0
    nrows_g=34 nrows=34 sparsity=.8599 nnzs=994, refcount: 7>
  <dData1D:(new from dSpData1D) n=994, refcount: 1>
refcount: 1>
new_DM -- step:     1
Initializing Density Matrix...
DM filled with atomic data:
<dSpData2D:DM initialized from atoms
  <sparsity:sparsity for geom step 0
    nrows_g=34 nrows=34 sparsity=.8599 nnzs=994, refcount: 8>
  <dData2D:DM n=994 m=1, refcount: 1>
refcount: 1>
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:       6      41

InitMesh: MESH = 30 x 90 x 48 = 129600
InitMesh: Mesh cutoff (required, used) =   200.000   223.865 Ry
New grid distribution [1]: sub = 2

stepf: Fermi-Dirac step function

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -198.917938
siesta: Eions   =      1621.026762
siesta: Ena     =       347.673774
siesta: Ekin    =       733.086614
siesta: Enl     =      -133.328654
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =       -25.529305
siesta: DUscf   =         4.461772
siesta: DUext   =         0.000000
siesta: Ex      =      -204.505585
siesta: Ec      =       -25.367744
siesta: Exc     =      -229.873328
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -916.616359
siesta: Etot    =      -924.535888
siesta: FreeEng =      -924.535888

        iscf     Eharris(eV)        E_KS(eV)     FreeEng(eV)     dDmax    Ef(eV) dHmax(eV)
   scf:    1     -916.616359     -924.535888     -924.535888  0.592950 -8.451626  4.380668
timer: Routine,Calls,Time,% = IterSCF        1       0.036   4.67
   scf:    2     -927.070557     -926.069260     -926.069260  0.123527 -7.868711  1.347524
   scf:    3     -926.276331     -926.236340     -926.236340  0.060948 -7.679048  0.244780
   scf:    4     -926.243199     -926.240154     -926.240154  0.004797 -7.729094  0.103523
   scf:    5     -926.243778     -926.242038     -926.242038  0.003432 -7.766399  0.068873
   scf:    6     -926.243473     -926.242785     -926.242785  0.002704 -7.790557  0.028490
   scf:    7     -926.242907     -926.242848     -926.242848  0.000677 -7.792173  0.012106
   scf:    8     -926.242870     -926.242859     -926.242859  0.000318 -7.790816  0.005226
   scf:    9     -926.242863     -926.242862     -926.242862  0.000108 -7.789294  0.002759
   scf:   10     -926.242863     -926.242862     -926.242862  0.000052 -7.788327  0.001620
   scf:   11     -926.242863     -926.242862     -926.242862  0.000044 -7.787787  0.001333
   scf:   12     -926.242863     -926.242863     -926.242863  0.000035 -7.787483  0.000876

SCF Convergence by DM+H criterion
max |DM_out - DM_in|         :     0.0000350534
max |H_out - H_in|      (eV) :     0.0008760983
SCF cycle converged after 12 iterations

Using DM_out to compute the final energy and forces
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:       6      41

siesta: E_KS(eV) =             -926.2429

siesta: E_KS - E_eggbox =      -926.2429

siesta: Atomic forces (eV/Ang):
----------------------------------------
   Tot    0.000000   -0.000000   -0.000000
----------------------------------------
   Max    2.636819
   Res    1.098431    sqrt( Sum f_i^2 / 3N )
----------------------------------------
   Max    2.636819    constrained

Stress tensor Voigt[x,y,z,yz,xz,xy] (kbar):     -150.73      -31.92       -1.52       -0.00       -0.00       -0.00
(Free)E + p*V (eV/cell)     -920.4954
Target enthalpy (eV/cell)     -926.2429

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -235.182818
siesta: Eions   =      1621.026762
siesta: Ena     =       347.673774
siesta: Ekin    =       692.346070
siesta: Enl     =      -121.809038
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =        -1.885664
siesta: DUscf   =         1.511615
siesta: DUext   =         0.000000
siesta: Ex      =      -197.998350
siesta: Ec      =       -25.054508
siesta: Exc     =      -223.052858
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -926.242863
siesta: Etot    =      -926.242863
siesta: FreeEng =      -926.242863

siesta: Final energy (eV):
siesta:  Band Struct. =    -235.182818
siesta:       Kinetic =     692.346070
siesta:       Hartree =     634.424529
siesta:         Edftu =       0.000000
siesta:       Eso     =       0.000000
siesta:    Ext. field =       0.000000
siesta:         Exch. =    -197.998350
siesta:         Corr. =     -25.054508
siesta:     Bulk bias =       0.000000
siesta:   Exch.-corr. =    -223.052858
siesta:  Ion-electron =   -1904.151435
siesta:       Ion-ion =    -125.809169
siesta:       Ekinion =       0.000000
siesta: D3 dispersion =       0.000000
siesta:         Total =    -926.242863
siesta:         Fermi =      -7.787483

siesta: Atomic forces (eV/Ang):
siesta:      1   -0.000000   -2.636819    0.000000
siesta:      2    0.429788    1.329793   -0.000000
siesta:      3   -0.429788    1.329793   -0.000000
siesta:      4   -0.000000    2.636819    0.000000
siesta:      5    0.429788   -1.329793   -0.000000
siesta:      6   -0.429788   -1.329793   -0.000000
siesta: ----------------------------------------
siesta:    Tot    0.000000   -0.000000   -0.000000

siesta: Stress tensor (static) (eV/Ang**3):
siesta:    -0.094076   -0.000000   -0.000000
siesta:    -0.000000   -0.019925    0.000000
siesta:    -0.000000   -0.000000   -0.000948

siesta: Cell volume =        150.000000 Ang**3

siesta: Pressure (static):
siesta:                Solid            Molecule  Units
siesta:           0.00041732          0.00031313  Ry/Bohr**3
siesta:           0.03831637          0.02875060  eV/Ang**3
siesta:          61.38958763         46.06353753  kBar
(Free)E+ p_basis*V_orbitals  =        -925.012809
(Free)Eharris+ p_basis*V_orbitals  =        -925.012809

siesta: Electric dipole (a.u.)  =   -0.000000   -0.000000    0.000000
siesta: Electric dipole (Debye) =   -0.000000   -0.000000    0.000000

cite: Please see "filter_cutoff.bib" for an exhaustive BiBTeX file.
cite: Please clearly indicate Siesta version in published work: 5.0.0-alpha-123-gfb8fd7d02
cite: This calculation has made use of the following articles
cite: which are encouraged to be cited in a published work.
        Primary SIESTA paper
          DOI: www.doi.org/10.1088/0953-8984/14/11/302


timer: Elapsed wall time (sec) =       1.399
timer: CPU execution times (sec):

Routine            Calls   Time/call    Tot.time        %
siesta                 1       1.392       1.392   100.00
Setup                  1       0.105       0.105     7.56
bands                  1       0.000       0.000     0.00
KSV_init               1       0.000       0.000     0.00
IterGeom               1       1.286       1.286    92.39
geom_init              1       0.623       0.623    44.71
state_init             1       0.048       0.048     3.46
hsparse                1       0.000       0.000     0.03
overlap                1       0.048       0.048     3.41
Setup_H0               1       0.539       0.539    38.71
naefs                  2       0.000       0.000     0.03
MolMec                 2       0.000       0.000     0.00
kinefsm                2       0.023       0.046     3.33
nlefsm                 2       0.204       0.409    29.34
DHSCF_Init             1       0.086       0.086     6.15
DHSCF1                 1       0.007       0.007     0.50
INITMESH               1       0.000       0.000     0.00
DHSCF2                 1       0.079       0.079     5.65
REMESH                 1       0.005       0.005     0.32
REORD                 36       0.000       0.003     0.22
PHION                  1       0.060       0.060     4.30
COMM_BSC              59       0.000       0.004     0.26
POISON                15       0.003       0.047     3.40
fft                   30       0.001       0.034     2.41
setup_H               13       0.036       0.462    33.19
DHSCF                 14       0.049       0.692    49.73
DHSCF3                14       0.035       0.497    35.67
rhoofd                14       0.014       0.189    13.59
XC                    14       0.008       0.113     8.14
GXC-CellXC            14       0.008       0.109     7.83
gridxc@cellXC         14       0.008       0.109     7.83
vmat                  14       0.009       0.132     9.49
IterSCF               12       0.036       0.430    30.86
compute_dm            12       0.000       0.002     0.13
diagon                12       0.000       0.002     0.13
r-eigvec              12       0.000       0.001     0.10
r-buildHS             12       0.000       0.000     0.00
rdiag                 12       0.000       0.001     0.10
rdiag1                12       0.000       0.000     0.01
rdiag2                12       0.000       0.000     0.01
rdiag3                12       0.000       0.001     0.07
rdiag4                12       0.000       0.000     0.00
r-buildD              12       0.000       0.000     0.02
MIXER                 11       0.000       0.000     0.01
PostSCF                1       0.234       0.234    16.80
DHSCF4                 1       0.196       0.196    14.06
dfscf                  1       0.182       0.182    13.10
overfsm                1       0.001       0.001     0.04
state_analysis         1       0.000       0.000     0.01
siesta_move            1       0.000       0.000     0.00
Analysis               1       0.000       0.000     0.03
optical                1       0.000       0.000     0.00
  

>> End of run:  28-SEP-2023  15:40:26
Job completed
